library/src/main/java/uk/co/samuelwall/materialtaptargetprompt/extras/PromptUtils.java

```diff
@@ -290,30 +289,27 @@ public static void scale(final PointF origin, final RectF base, final RectF out,
      * @param layout The layout to check.
      * @return True if the text in the supplied layout is displayed right to left. False otherwise.
      */
-    public static boolean isRtlText(final Layout layout, final Resources resources)
+    public static boolean isRtlText(@Nullable final Layout layout, @NonNull final Resources resources)
     {
         boolean result = false;
         if (layout != null)
         {
             // Treat align opposite as right to left by default
             result = layout.getAlignment() == Layout.Alignment.ALIGN_OPPOSITE;
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.ICE_CREAM_SANDWICH)
```

Comment:
`ICE_CREAM_SANDWICH == 14` which is the min sdk of this lib. Too bad there's not a lint warning for this.